### ISSUE TITLE
Execute onchange inside a child container 

### ISSUE THREAD

**Issue by caiquecaleiro** [erikras/redux-form#2297]
Hello there, I'm new on React/Redux and I'd like to know whether it's possible to execute a Field onchange event on a Child component.

I have three containers in a feature, which the first one has the Redux-form, which two Field (description, status):
```javascript
export default connect(mapStateToProps, mapDispatchToProps)(reduxForm({
  form: 'InterestAreaForm',
  validate
})(InterestArea));
```
The other two containers have a Filter with a form that is used to filter the last container, a Grid.
So, whenever the user selects a row on the grid and clicks on Edit button, I'd like to change the values of the Description and Status fields based on the row data.

How can I make these components communicate?

Thank you for your attention.

**Comment by davincho** [comment]
Have a look at http://redux-form.com/6.3.1/docs/api/ActionCreators.md/ `change(form:String, field:String, value:any)` to change a field of any arbitrary form.

Your redux form component gets passed all props listed at http://redux-form.com/6.3.1/docs/api/Props.md/. So within form component you can change a field with `change(field:String, value:any)`. Of course you could pass down this function to any child component.

**Comment by caiquecaleiro** [comment]
@davincho thanks for your answer. I'm aware about the change, but I was worried about the structure of this feature:
- I have got a Form to create new records, which I use it inside a container.
- I have got another container with Filter and a Grid components. Whenever the user clicks on Filter, it will render the new data on Grid and whenever he clicks on a row, he can edit it in the first container (form).

So in this case, is it right to consider that the Form container is the parent of the Filter/Grid container and then pass the change for these children?
